cmd/kyma/alpha/install/cmd.go

```diff
@@ -40,6 +40,10 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd.Flags().StringVarP(&o.OverridesYaml, "overrides", "o", "", "Path to a YAML file with parameters to override.")
 	cobraCmd.Flags().StringVarP(&o.ComponentsYaml, "components", "c", "", "Path to a YAML file with component list to override.")
 	cobraCmd.Flags().StringVarP(&o.ResourcesPath, "resources", "r", "", "Path to Kyma resources folder.")
+	cobraCmd.Flags().DurationVarP(&o.CancelTimeout, "cancel-timeout", "", 900*time.Second, "Time after which the workers' context is canceled. Pending worker goroutines (if any) may continue if blocked by a Helm client.")
+	cobraCmd.Flags().DurationVarP(&o.QuitTimeout, "quit-timeout", "", 1200*time.Second, "Time after which the installation is aborted. Worker goroutines may still be working in the background. This value must be greater than the value for cancel-timeout.")
+	cobraCmd.Flags().IntVar(&o.HelmTimeout, "helm-timeout", 360, "Timeout for the underlying Helm client (in seconds).")
```

Comment:
Shouldn't this also be a `time.Duration` to be consistent with the other flags?
We can then convert it to satisfy the installation package format.